src/qml/CMakeLists.txt

```diff
@@ -63,6 +63,4 @@ list(
   ${qfield_qmlfiles_theme_extra}
   ${qfield_qmlfiles_qfieldcontrols})
 
-set(qfield_qmlfiles
-    ${qfield_qmlfiles}
-    PARENT_SCOPE)
+set_target_properties(qfield_qml PROPERTIES QML_FILES "${qfield_qmlfiles}")
```

Comment:
`set_property(TARGET qfield_qml PROPERTY QML_FILES ${qfield_qmlfiles})` will preserve the list as list

Reply:
@m-kuhn , done.